libraries/AP_SerialManager/AP_SerialManager.h

```diff
@@ -63,6 +63,12 @@
 #define AP_SERIALMANAGER_SToRM32_BUFSIZE_RX     128
 #define AP_SERIALMANAGER_SToRM32_BUFSIZE_TX     128
 
+// Aerotenna uLanding Altimeter
+// Note that size of UART FIFO is 128 for Altera-OcPoc board
+#define AP_SERIALMANAGER_ULANDING_BAUD           115200
+#define AP_SERIALMANAGER_ULANDING_BUFSIZE_RX     128
+#define AP_SERIALMANAGER_ULANDING_BUFSIZE_TX     128
```

Comment:
What does the 128 byte tx buffer do? Seems like nothing. Can this be safely changed to.. 1?